packages/broker/test/integration/plugins/operator/VoteOnSusppectNodeService.test.ts

```diff
@@ -0,0 +1,131 @@
+
+    })
+    
+    it('allows to flag an operator as malicious', async () => {
+        const flagger = await createWalletAndDeployOperator(provider, config, theGraphUrl, ADMIN_PRIV_KEY)
+        logger.debug('deployed flagger contract ' + flagger.operatorConfig.operatorContractAddress)
+        const target = await createWalletAndDeployOperator(provider, config, theGraphUrl, ADMIN_PRIV_KEY)
+        logger.debug('deployed target contract ' + target.operatorConfig.operatorContractAddress)
+        const voter = await createWalletAndDeployOperator(provider, config, theGraphUrl, ADMIN_PRIV_KEY)
+        logger.debug('deployed voter contract ' + voter.operatorConfig.operatorContractAddress)
+
+        await new Promise((resolve) => setTimeout(resolve, 5000)) // wait for events to be processed
+        const flaggerClient: MockProxy<StreamrClient> = mock<StreamrClient>()
+        const flaggerVoteService = new VoteOnSuspectNodeService(flaggerClient, flagger.operatorConfig)
+        await flaggerVoteService.start()
+
+        const targetClient: MockProxy<StreamrClient> = mock<StreamrClient>()
+        const targetVoteService = new VoteOnSuspectNodeService(targetClient, target.operatorConfig)
+        await targetVoteService.start()
+        expect(targetClient.subscribe.mock.calls.length).toBe(0)
+
+        const voterClient: MockProxy<StreamrClient> = mock<StreamrClient>()
+        const voterVoteService = new VoteOnSuspectNodeService(voterClient, voter.operatorConfig)
+        await voterVoteService.start()
+
+        logger.debug('deploying sponsorship contract')
+        const sponsorship = await deploySponsorship(config, adminWallet, {
+            streamId: streamId1 })
+        logger.debug('sponsoring sponsorship contract')
+        await (await token.connect(flagger.operatorWallet).approve(sponsorship.address, parseEther('500'))).wait()
+        await (await sponsorship.connect(flagger.operatorWallet).sponsor(parseEther('500'))).wait()
+
+        logger.debug('each operator delegates to its operactor contract')
+        logger.debug('delegating from flagger: ' + flagger.operatorWallet.address)
+        await (await token.connect(flagger.operatorWallet).transferAndCall(flagger.operatorContract.address,
+            parseEther('200'), flagger.operatorWallet.address)).wait()
+        logger.debug('delegating from target: ' + target.operatorWallet.address)
+        await (await token.connect(target.operatorWallet).transferAndCall(target.operatorContract.address,
+            parseEther('200'), target.operatorWallet.address)).wait()
+        logger.debug('delegating from voter: ' + voter.operatorWallet.address)
+        await (await token.connect(voter.operatorWallet).transferAndCall(voter.operatorContract.address,
+            parseEther('200'), voter.operatorWallet.address)).wait()
+        
+        logger.debug('staking to sponsorship contract from flagger and target and voter')
```

Comment:
I won't add a Github comment on each, let's just make the rest of these `debug` level statements in the test `trace`.